pkg/injector/pod_patch.go

```diff
@@ -98,6 +101,15 @@ func (i *injector) getPodPatchOperations(ar *v1.AdmissionReview,
 	// Projected volume with the token
 	tokenVolume := sidecar.GetTokenVolume()
 
+	// Pod annotations
+	podPatchOps := []common.PatchOperation{
```

Comment:
Should this be empty if `GetAppPodAnnotations` returns an object with len == 0?

Reply:
Yes, changed!